configdialog/lxqtpageselectwidget.cpp

```diff
@@ -102,12 +103,14 @@ QSize PageSelectWidget::sizeHint() const
     int w = 0;
     for(int i=0; i< count(); ++i)
     {
-        QRect rect = fontMetrics().boundingRect(QRect(), Qt::AlignLeft | Qt::TextWordWrap, item(i)->text());
+        QRect rect = fontMetrics().boundingRect(QRect(), Qt::AlignHCenter | Qt::TextWordWrap, item(i)->text());
         w = qMax(w, rect.width());
     }
+    // consider the icon size too
+    w = qMax(w, QApplication::style()->pixelMetric(QStyle::PM_IconViewIconSize));
```

Comment:
Do you see any trace of `QWidget *w` in `qstylesheetstyle.cpp` → `QStyleSheetStyle::pixelMetric` → `case PM_ScrollBarExtent:`? It's just in `QRenderRule rule = renderRule(w, opt);` before `switch`.

Anyway, if you worry that something may be lost, add it -- it'll neither make any difference nor do any harm, AFAIK.